src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -476,7 +519,7 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 			return undefined;
 		}
 
-		return getNWords(element.response.value, numWordsToRender);
+		return getNWords(element.response.asString(), numWordsToRender);
```

Comment:
Seems like the tree won't be rendered during progressive rendering? With your test, that may not have happened since the tree data comes at the end after a delay, but depending on the timing progressive rendering could happen while there is some tree data

Reply:
Hmm, this doesn't seem to happen even if I resolve the `resolveContent` promise immediately, let me investigate